osu.Game.Rulesets.Osu/Skinning/Argon/ArgonMainCirclePiece.cs

```diff
@@ -123,14 +120,18 @@ protected override void LoadComplete()
 
                 // Accent colour may be changed many times during a paused gameplay state.
                 // Schedule the change to avoid transforms piling up.
-                Scheduler.AddOnce(updateStateTransforms);
+                Scheduler.AddOnce(() =>
+                {
+                    updateStateTransforms(drawableObject, drawableObject.State.Value);
+
+                    ApplyTransformsAt(double.MinValue, true);
+                    ClearTransformsAfter(double.MinValue, true);
```

Comment:
Weirded out by this. So first, the new accent colour is applied by `updateStateTransforms()`, and... *after* that transforms are rewound and cleared? How does that work? `DrawableHitObject` does it in the inverse order, does it not? What am I missing?

Reply:
Yeah this does seem back-to-front. Oops.